source/reference/connection-string.txt

```diff
@@ -130,8 +130,8 @@ Examples
            your deployment when you click the :atlas:`Connect
            </connect-to-database-deployment/>` button in the Atlas UI. 
            To register a new Atlas account, start at the 
-           :mdbacct:`MongoDB Atlas registration page 
-           </register?tck=docs_server>`. To learn more, see
+           `MongoDB Atlas registration page 
+           <https://www.mongodb.com/cloud/atlas/register?tck=docs_server_connection_cta>`_. To learn more, see
```

Comment:
```suggestion
           <https://www.mongodb.com/cloud/atlas/register?tck=docs_server_connection_cta>`__. To learn more, see
```

Reply:
@corryroot I believe external links need two underscores. I'll fix here and merge.